columnar/src/column_values/u64_based/blockwise_linear.rs

```diff
@@ -63,7 +63,10 @@ impl BlockwiseLinearEstimator {
         if self.block.is_empty() {
             return;
         }
-        let line = Line::train(&VecColumn::from(&self.block));
+        let column = VecColumn::from(std::mem::take(&mut self.block));
```

Comment:
why was this needed? I don't understand.

Reply:
Extending the `ColumnValues` trait with another trait (any trait, even an empty one) requires lifetimes to be `'static`. This requirement is no problem when working on the values instead of references, so I changed `VecColumn` to a owned version.